pkg/diskmaker/controllers/lvset/reconcile.go

```diff
@@ -165,6 +179,24 @@ DeviceLoop:
 				continue DeviceLoop
 			}
 		}
+
+		// skip devices younger than deviceMinAge
+		if isOldEnough {
```

Comment:
but now this function and variable reads weird. It says - `if isOldEnough` which actually signals the opposite that the device is actually younger! can the function and variable be named appropriately?

Reply:
not sure I follow: `isOldEnough` signals that the device is older than `minDeviceAge`

Reply:
is it the comment?

Reply:
I get it, I'll do 
```
if !isOldEnough {
   continue DeviceLoop
}
```